docs/content/anonymization/core-anonymizers.md

```diff
@@ -307,6 +307,69 @@ customer:
 ```
 :::
 
+## DateAnonymizer
+
+Anonymize dates by either:
+- randomly choosing an date or datetime in a given range delimited by `min` and `max` options,
+- altering the initial value by adding it a random value picked in a range computed from the `delta` options.
+
+`min` and `max` options can be any string that can be parsed as a date by the `DateTime`
+class constructor, for example:
+ - an absolute date: `2024-03-15` or datetime: `2024-03-15 10:28:56`,
+ - a relative time: `now +2 hours`, `-3 month`, ...
+
+`delta` option can be either:
+ - an ISO interval specification, such as: `P1DT1M` (1 day and 1 minute),
+ - a human readable date string that PHP can parse: `1 month -3 day +3 minutes`.
+
+Giving invalid values for options will raise an exception before running the anonymization.
```

Comment:
Don't know if you should put this here, it will be kind of true for all anonymizers after all